data/json/furniture_and_terrain/terrain-floors-outdoors.json

```
@@ -201,6 +201,25 @@
       "items": [ { "item": "rock", "count": [ 2, 10 ] }, { "item": "rebar", "count": [ 0, 4 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_zebra",
+    "alias": [ "t_zebra_h", "t_zebra_v" ],
+    "name": "pedestrian crossing",
+    "connects_to": "PAVEMENT",
+    "description": "White polymer lines painted on the pavement which indicate places where pedestrians should cross the road.",
+    "symbol": "-",
```

```suggestion
    "symbol": ".",
```
The `-` kinda mkes them look like obstacles, I think it feels visually confusing.